aws/networking/main.tf

```diff
@@ -0,0 +1,79 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+resource "aws_vpc" "main" {
+  cidr_block           = "10.0.0.0/16"
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+  tags = local.tags
+}
+
+resource "aws_internet_gateway" "main" {
+  vpc_id = aws_vpc.main.id
+  tags = local.tags
+}
+
+resource "aws_egress_only_internet_gateway" "main" {
+  vpc_id = aws_vpc.main.id
+  tags = local.tags
+}
+
+resource "aws_route_table" "main" {
+  vpc_id = aws_vpc.main.id
+
+  route {
+    cidr_block = "0.0.0.0/0"
+    gateway_id = aws_internet_gateway.main.id
+  }
+
+  route {
+    ipv6_cidr_block        = "::/0"
+    egress_only_gateway_id = aws_egress_only_internet_gateway.main.id
+  }
+  tags = local.tags
+}
+
+resource "aws_main_route_table_association" "default" {
+  vpc_id         = aws_vpc.main.id
+  route_table_id = aws_route_table.main.id
+}
+
+resource "aws_subnet" "primary" {
+  vpc_id            = aws_vpc.main.id
+  cidr_block        = "10.0.1.0/24"
+  availability_zone = data.aws_availability_zones.available.names[0]
+  tags = local.tags
+}
+
+resource "aws_subnet" "secondary" {
```

Comment:
Similarly[^1], let's rename this to `private_secondary`.

[^1]: https://github.com/withlogicco/terraform-modules/pull/4/files#diff-6fae935ba08a96dd1f84b78d3f7245778fc816c0d17305d731b8489cb5ceaa08R42